src/test/java/com/google/api/codegen/testdata/discoveries/foo.v1.json.overrides

```diff
@@ -4,5 +4,27 @@
   },
   "js|python": {
     "discoveryDocUrl": "localhost:8080/$discovery/foo?version=v1"
+  },
+  "csharp|go|java": {
+    "methods": {
+      "foo.bar.get": {
+        "fields": {
+          "foobar": {
+            "defaultValue": "\" \""
```

Comment:
Specifically, for array fields, if a default value is specified, that element is added to the array. So if no override is specified, as usual, this is what shows up:

```
List<Foo> foos = new List<Foo>();
```

If an override is applied (in this case, `"defaultValue": "bar"`), this shows up:
```
List<Foo> foos = new List<Foo>();
foos.add("bar");
```